src/views/blocks/rsvp/form/quantity.php

```diff
@@ -15,6 +15,6 @@
 		<?php $this->template( 'blocks/rsvp/form/quantity-plus' ); ?>
 	</div>
 	<span class="tribe-block__rsvp__number-input-label">
-		<?php echo esc_html__( 'RSVPs', 'event-tickets' ); ?>
+		<?php echo tribe_get_rsvp_label_plural( 'number_input_label' ); ?>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 3f6987b53c7a003169750e36215c7693</sub>